streams/dshr_stream_mod.F90

```diff
@@ -1531,39 +1584,34 @@ subroutine shr_stream_getCalendar(strm, k, calendar)
     character(*)                ,intent(out)   :: calendar ! calendar name
 
     ! local
-    type(ESMF_VM)          :: vm
-    integer                :: myid
     integer                :: vid, n
     character(CX)          :: fileName
     character(CL)          :: lcal
     integer(PIO_OFFSET_KIND) :: attlen
     integer                :: old_handle
     integer                :: rCode
-    integer :: rc
     character(*),parameter :: subName = '(shr_stream_getCalendar) '
     !-------------------------------------------------------------------------------
 
     lcal = ' '
     calendar = ' '
     if (k > strm%nfiles) call shr_sys_abort(subname//' ERROR: k gt nfiles')
 
-    call ESMF_VMGetCurrent(vm, rc=rc)
-    if (ChkErr(rc,__LINE__,u_FILE_u)) return
-
-    call ESMF_VMGet(vm, localPet=myid, rc=rc)
-    if (ChkErr(rc,__LINE__,u_FILE_u)) return
-
     fileName  = strm%file(k)%name
 
     if (.not. pio_file_is_open(strm%file(k)%fileid)) then
-       if(myid == 0) write(strm%logunit, '(a)') trim(subname)//' opening stream filename = '//trim(filename)
+       if (debug>0 .and. mainproc) then
+          write(logout,'(3x,2a)') trim(subname),' opening stream filename = '//trim(filename)
+       end if
        rcode = pio_openfile(strm%pio_subsystem, strm%file(k)%fileid, strm%pio_iotype, trim(filename))
-    else if(myid == 0) then
-       write(strm%logunit, '(a)') trim(subname)//' reading stream filename = '//trim(filename)
+    else
+       if (debug>0 .and. mainproc) then
+          write(logout,'(3x,2a)') trim(subname),' reading stream filename = '//trim(filename)
+       end if
     endif
 
     rCode = pio_inq_varid(strm%file(k)%fileid, 'time', vid)
-    if(vid .lt. 0) then
+    if (vid .lt. 0) then
```

Comment:
As long as you are modifying this line, please update logic syntax:
```suggestion
    if (vid < 0) then
```

Reply:
done